test/tstLinearBVH.cpp

```diff
@@ -285,37 +285,37 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(buffer_optimization, DeviceType,
 
   // optimal size
   BOOST_CHECK_NO_THROW(
-      bvh.query(ExecutionSpace{}, queries, indices, offset,
-                ArborX::Experimental::TraversalPolicy().setBufferSize(
-                    -max_results_per_query)));
+      ArborX::query(bvh, ExecutionSpace{}, queries, indices, offset,
```

Comment:
You didn't qualify the `query()` call above but you did here and below.  Please be consistent.  I would prefer not qualifying.

Reply:
Qualified query call above. Not fixing the rest.